test/tools/cli/IonProcess.java

```diff
@@ -61,66 +62,68 @@ public static void main(final String[] args) {
             printHelpTextAndExit(e.getMessage(), parser);
         }
 
+        //Use this variable for symbol_table event which changes ionWriter
+        IonWriter ionWriterForOutput = null;
```

Comment:
See my commend below about adding the IonWriter to the ProcessContext. Currently this doesn't have any effect. You'll probably also have to remove the IonWriter instantiation from the try-with-resources and add a `finally` block in which you manually close it, since the reference may change during execution.

Reply:
Changed